spec/lib/terraforming/resource/iam_group_spec.rb

```diff
@@ -47,36 +59,38 @@ module Resource
         end
       end
 
-      describe ".tfstate" do
-        it "should generate tfstate" do
-          expect(described_class.tfstate(client: client)).to eq({
-            "aws_iam_group.hoge" => {
-              "type" => "aws_iam_group",
-              "primary" => {
-                "id" => "hoge",
-                "attributes" => {
-                  "arn"=> "arn:aws:iam::123456789012:group/hoge",
-                  "id" => "hoge",
-                  "name" => "hoge",
-                  "path" => "/",
-                  "unique_id" => "ABCDEFGHIJKLMN1234567",
+      describe '.tfstate' do
+        it 'should generate tfstate' do
+          expect(described_class.tfstate(client: client)).to eq(
+            {
+              'aws_iam_group.hoge' => {
+                'type' => 'aws_iam_group',
+                'primary' => {
+                  'id' => 'hoge',
+                  'attributes' => {
+                    'arn' => 'arn:aws:iam::123456789012:group/hoge',
+                    'id' => 'hoge',
+                    'name' => 'hoge',
+                    'path' => '/',
+                    'unique_id' => 'ABCDEFGHIJKLMN1234567',
+                  }
                 }
-              }
-            },
-            "aws_iam_group.fuga" => {
-              "type" => "aws_iam_group",
-              "primary" => {
-                "id" => "fuga",
-                "attributes" => {
-                  "arn"=> "arn:aws:iam::345678901234:group/fuga",
-                  "id" => "fuga",
-                  "name" => "fuga",
-                  "path" => "/system/",
-                  "unique_id" => "OPQRSTUVWXYZA8901234",
+              },
+              'aws_iam_group.fuga' => {
+                'type' => 'aws_iam_group',
+                'primary' => {
+                  'id' => 'fuga',
+                  'attributes' => {
+                    'arn' => 'arn:aws:iam::345678901234:group/fuga',
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.